datamodel/src/main/kotlin/com/hartwig/actin/datamodel/clinical/SequencingTest.kt

```diff
@@ -39,6 +39,17 @@ data class SequencedSkippedExons(val gene: String, val exonStart: Int, val exonE
     }
 }
 
+data class SequencedVirus(val virus: SequencedVirusInput)
+
+enum class SequencedVirusInput {
+    EBV,
+    HBV,
+    HHV_8,
+    HPV_HIGH_RISK,
+    HPV_LOW_RISK,
+    MCV
+}
```

Comment:
Given these definitions, I think you could do without the `SequencedVirus` class and just store a `Set<SequencedVirusInput>` in the `SequencingTest`. However, I think it would be better to reference the existing `VirusType` enum and just store a `DriverLikelihood` or boolean here to capture the same information:
```suggestion
data class SequencedVirus(
    val type: VirusType,
    val isLowLikelihood: Boolean = false
)
```